testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -244,3 +248,26 @@ def test_PDB_metals():
     assert u.atoms[1].mass == pytest.approx(tables.masses["FE"])
     assert u.atoms[2].mass == pytest.approx(tables.masses["CA"])
     assert u.atoms[3].mass == pytest.approx(tables.masses["MG"])
+
+
+def test_PDB_elements():
+    from MDAnalysis.topology import tables
+
+    u = mda.Universe(PDB_elements, format='PDB')
+    element_list = ['N', 'C', 'C', 'O', 'C', 'C', 'O', 'N', 'H', 'H', 'H',
+                    'H','H','H', 'H', 'H', 'CU', 'FE', 'C', 'MG']
+    assert len(u.atoms.elements) == len(element_list)
+    assert all(i == j for i,j in zip(element_list, u.atoms.elements))
+
+
+
+def test_PDB_elements_warnings():
+    from MDAnalysis.topology import tables
+
+    u = mda.Universe(PDB_wrong_ele, format='PDB')
+    with pytest.warns(UserWarning) as record:
+        warnings.warn("Element record found to be non-physical. Guessing element from atom name: CGA", UserWarning)
```

Comment:
That's not quite how pytest.warns works. You probably want this:
```python
with pytest.warns(UserWarning) as record:
    u = mda.Universe(PDB_wrong_ele)
```

In this case you yourself raise the warning you are looking for.

Reply:
Oh, my bad. Will correct this